mask/eolearn/mask/cloud_mask.py

```diff
@@ -156,15 +156,13 @@ def _downscaling(self, hr_array, meta_info, interp='linear', smooth=True):
         if (self.cm_size_y is None) and (self.cm_size_x is None):
             return hr_array, None
 
-        # Rescaling factor in spatial (height, width) dimensions
+        # Rescaling factor in spatial (width, height) dimensions
```

Comment:
Why is order of dimensions `(width, height)` and not `(height, width)`? Aren't images in EOPatches saved as `(height, width, bands)`?

Reply:
See answer above.